CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
```

Comment:
As previous 1.44.0 release went with squash merge, many of the commits are missing in `main` branch. As the plugin captured the diffs b/w `main` branch & release branch commits, changelog has duplicate updates - clean them up